src/Instances/defaultProps.lua

```diff
@@ -5,6 +5,22 @@
 	the New function.
 ]]
 
+local superclassProps = {
```

Comment:
In line with the formatting of the rest of the file, it's preferred to copy these definitions into the table directly. This is for two reasons:

- if a user needs to refer to this file to find default properties for a given class name, it's easier for them to locate one class name rather than two
- if a subclass has extra properties in the future, it becomes awkward to reconcile

We may address the duplication here in other ways if it's shown to be a problem, but for the time being, it's better to follow the conventions of the surrounding code.

Reply:
Oh okay, I only did it like that because of how many instances inherited from superclasses and didn't want to repeat code, but I'll change!